samples/bluetooth/direction_finding_connectionless_tx/README.rst

```diff
@@ -46,6 +54,8 @@ To build this sample with angle of arrival mode only, set ``OVERLAY_CONFIG`` to
 See :ref:`cmake_options` for instructions on how to add this option.
 For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
 
+To build this sample with angle of arrival mode only for :ref:`nRF5340 DK <ug_nrf5340>` add content of :file:`overlay-aoa.conf` to :file:`child_image/hci_rpmsg.conf`.
```

Comment:
Make this the second sentence in the section
```suggestion
To build this sample for :ref:`nRF5340 DK <ug_nrf5340>`, with angle of arrival mode only, add content of :file:`overlay-aoa.conf` file to :file:`child_image/hci_rpmsg.conf` file.
```

Reply:
Updated according to suggestion